setup-services.sh

```diff
@@ -147,7 +150,7 @@ $SUDO $ZYPPER install \
   cd $MYDIR/rust
   cargo build
 
-  ln -st /usr/bin $MYDIR/rust/target/debug/agama{,*server}
+  sudo ln -sft /usr/bin $MYDIR/rust/target/debug/agama{,*server}
```

Comment:
and same here `https://github.com/agama-project/agama/pull/1687/files#diff-327a0d405c7c05ecb8395935997d54a646315e7da5e275748c565467fc3b7f61R17` to allow to use different way to have root permission